ministryofjustice/laa-apply-for-criminal-legal-aid
# Upgrade gov uk components gem

## Description of change
Upgrades gov uk components gem which in turn updates the govuk logo 

## Link to relevant ticket

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

### After changes:
<img width="992" alt="Screenshot 2024-02-28 at 17 18 56" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/6cc2622b-b32a-44ae-9150-506f3dbb8c44">

## How to manually test the feature
Pull down branch and run bundle install. Then rebuild your assets with `rails assets:clobber` and `rails dartsass:build`. Restart your server and then check the crown has changed. Also check whether there have been regressions in other parts of the app
